package.json

```diff
@@ -6,8 +6,8 @@
     "node": ">= 10.13"
   },
   "scripts": {
-    "prettier": "prettier --write '/code/static/**/*.js'",
-    "prettier-ci": "prettier --list-different '/code/static/**/*.js' || (echo '\n\nThis failure means you did not run `make format` before committing\n\n' && exit 1)"
+    "prettier": "prettier --write '/code/**/*.js'",
+    "prettier-ci": "prettier --list-different '/code/**/*.js' || (echo '\n\nThis failure means you did not run `make format` before committing\n\n' && exit 1)"
```

Comment:
Any reason to not use `prettier -c ...`

Reply:
Probably didn't exist when we added that in addons-frontend. Thanks!